testing/kubernetes/bookinfo.yaml

```diff
@@ -53,18 +53,16 @@ spec:
         env:
         - name: A8_PROXY
           value: "true"
-        - name: A8_CONTROLLER_URL
-          value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_DISCOVERY_ADAPTER
           value: kubernetes
+        - name: A8_RULES_ADAPTER
+          value: kubernetes
         - name: A8_KUBERNETES_POD_NAME
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
         - name: A8_ELASTICSEARCH_SERVER
           value: $(LOGSERVER_SERVICE_HOST):9200
-        - name: A8_CONTROLLER_POLL
-          value: 5s
```

Comment:
Please see #514 for changed deployment files, integrating Shriram's comments as well.
I've ran through the entire walkthrough which can be used to confirm the testing scripts, if needed.